modules/supporting_encapsulation.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-cpu-usage.adoc
+
+[id="supporting-encapsulation_{context}"]
+= Supporting encapsulation
+
+As an OpenShift support engineer or developer, you want to inspect Kubernetes-specific mountpoints.
+
+== Locating the encapsulated namespaces
+
+Any monitoring tool that relies on the ability to run in the host OS and have visibility of mountpoints created by Kubelet, the Container Runtime, or containers themselves, will need to enter the container mount namespace in order to see these mountpoints.
+
+[source,yaml]
+----
+  sh-4.4# readlink /proc/$(pgrep kubelet)/ns/mnt
+
+  sh-4.4# readlink /proc/$(pgrep crio)/ns/mnt
+----
+
+
+== Inspecting the encapsulated namespaces
+
+An administrator logged in to the host OS wishing to obtain information about the Kubernetes system for debugging or auditing purposes will need to be aware of the new mount namespace, and will need to run `nsenter` to view or interact with these mountpoints.
+
+Issue the following command:
+[source,yaml]
+----
+$ nsenter --mount=/run/container-mount-namespace/mnt ... <parameters ...> ...
```

Comment:
This isn't right; it refers to the old implementation.  Will update on Monday.

Reply:
```suggestion
$ kubensenter <parameters ...> ...
```

Reply:
Updated.